### PR TITLE
Added Duplicated Borrow

### LINKED ISSUE #156: Wrong message printed for deleting module
image

### PR DESCRIPTION
**Pull request by MuhammadHoze**
1. Added checks for duplicate book description

```
borrow soccer book date/10-11-2020
    ____________________________________________________________
     Got it. I've added this book: soccer book
	   (Loan Date: 10 Nov 2020)
	   (Due Date: 10 Dec 2020)
    ____________________________________________________________

borrow soccer book date/10-11-2020
    ____________________________________________________________
     ~Error~ Book already exists!
    ____________________________________________________________

```

2. Edited the link command to only allow four types of lesson "lecture/tutorial/lab/project" 

```
add link m/CS2113 t/le u/https://zoom.com
    ____________________________________________________________
     ~Error~ Please input a correct type of lesson:
 	 t/lecture
	 t/tutorial
	 t/lab
	 t/project
    ____________________________________________________________

```

3. Added the intro of my PPP 

4. Made some adjustments to the TOC of  UG, added flag notes and edited the welcome message 

5. Fixes #156   
      Changed message of delete module
```
delete module 1
    ____________________________________________________________
     Noted. I've removed this module:
       [CM][C+] CS2119 (4 MC) (AY2021S2)
     Now you have 0 module(s) in the list.
    ____________________________________________________________
```

### CODE REVIEW

File: src/main/java/seedu/duke/commands/ClearCommand.java [hunk 1]
```diff
@@ -25,6 +26,8 @@ public ClearCommand(String command) {
         this.description = command;
     }
 
+
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/commands/ClearCommand.java]: Why is there so much random whitespace

File: src/main/java/seedu/duke/commands/HelpCommand.java [hunk 2]
```diff
@@ -36,6 +36,7 @@ public void execute(Model model) throws DukeException {
                 + "\n\n     " + ReturnCommand.MESSAGE_USAGE
                 + "\n\n     " + SetCommand.MESSAGE_USAGE;
 
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/commands/HelpCommand.java:38]: Here

File: src/main/java/seedu/duke/model/itemlist/ItemList.java [hunk 3]
```diff
@@ -27,6 +29,7 @@
      */
     public ItemList(ArrayList<T> items) {
         this.items = items;
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/model/itemlist/ItemList.java:32]: Here

File: src/main/java/seedu/duke/storage/Storage.java [hunk 4]
```diff
@@ -168,11 +170,13 @@ public void save(ItemList items, String saveFilePath) throws DukeException {
         try {
             fw.write(saveString);
             fw.close();
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/storage/Storage.java:171]: Random whitespace

File: src/main/java/seedu/duke/storage/Storage.java [hunk 5]
```diff
@@ -168,11 +170,13 @@ public void save(ItemList items, String saveFilePath) throws DukeException {
...
         } catch (IOException e) {
             throw new DukeException(Messages.EXCEPTION_SAVE_FILE);
         }
     }
 
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/storage/Storage.java:177]: Whitespace

Comment (MuhammadHoze) [reply src/main/java/seedu/duke/storage/Storage.java:177]: yeap noted for all, removed them already as well as the Manifest file

**Pull request merged**